src/link/style.scss

```diff
@@ -364,29 +365,36 @@ body {
 					cursor: pointer;
 				}
 
-				&-change {
-
-					&::before {
-						position: relative;
-						content: url(./images/change.svg);
-						width: 20px;
-						height: 20px;
-						left: 14px;
+				&-option {
+					display: flex;
+					flex-direction: column;
+					align-items: center;
 
+					&-icon {
+						&-change {
+							width: 20px;
+							height: 20px;
+							margin-bottom: 5px;
+							content: url(./images/search-white.svg);
```

Comment:
I'm confused. To my knowledge `content:` only works on pseudo elements.

Reply:
Good catch, updating to `background-image`